packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts

```diff
@@ -5,6 +5,11 @@ import { IStyle, ITheme } from '../../Styling';
 import { IRefObject, IRenderFunction, IStyleFunctionOrObject } from '../../Utilities';
 
 export interface IChoiceGroup {
+  /**
+   * Gets the current checked option.
+   */
+  optionChecked: IChoiceGroupOption | undefined;
```

Comment:
>optionChecked [](start = 2, length = 13)

This is required prop. Does it mean all the consumers passing componentRef should now be passing an object which contains this prop too?

Reply:
🙄 I might have missed that))) And this is exactly why the reviews are important.

Reply:
Wait... this is not a prop... it's just an interface for the ChoiceGroup to implement. I will make sure it gets implemented by `ChoiceGroupBase` class instead.

Reply:
Hmmm it's already there:
https://github.com/OfficeDev/office-ui-fabric-react/blob/bc14f20836eed44160a53d367285cbd2110f752c/packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx#L17